submdspan/submdspan.md

```diff
@@ -567,6 +581,12 @@ namespace std {
     constexpr auto submdspan(
       const mdspan<ElementType, Extents, LayoutPolicy, AccessorPolicy>& src,
       SliceSpecifiers...slices) -> @_see below_@;
+
+  template<typename T>
+  concept @_integral-constant-like_@ =       // @_exposition only_@
+         std::is_integral_v<decltype(T::value)>
+      && !std::is_same_v<bool, std::remove_const_t<decltype(T::value)>>
+      && T() == T::value;
```

Comment:
Do we need to mention it in the wording and state in words that the underlying integral type cannot be `bool`?